dist/profile/templates/kubernetes/resources/evergreen/deployment.yaml.erb

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: evergreen
+spec:
+  replicas: 1
```

Comment:
Is it a stateless or statefull application? 
If it's a stateless app then I would increase the replicas to at least 3 in order to bring redundancy and the possibility to do  AB deployment.